script.js

```diff
@@ -0,0 +1,58 @@
+//[ I ] Порахуй голосні!
+function countVowels(str) {
+    //зробив масив з голосними літерами, з яким буду звірятись
+    let arrVowels = ["а","е","є","и","і","і","ї","о","у","ю","я","a","e","i","o","u","y", ]
+    let count = 0; //додав зміну для підрахунку
+    for (let char of str){
+        let newChar = char.toLowerCase();//всі літери переводжу в ніжній регістр

```

Comment:
я б краще не окремо кожну букву переводив, а одразу всю стрінгу )
```suggestion
    for (let char of str.toLowerCase()){
        let newChar = char;//всі літери переводжу в ніжній регістр

```